zuko/bayesian.py

```diff
@@ -0,0 +1,391 @@
+r"""Utils for bayesian flows."""
+
+from __future__ import annotations
+
+__all__ = [
+    "BayesianModel",
+]
+
+import copy
+import math
+import torch
+import torch.nn as nn
+import warnings
+
+from contextlib import contextmanager
+
+from .nn import Linear, MaskedLinear, linear
+
+
+def _softclip(x, extreme_val=11.0):
+    """Soft clipping function to keep x in [extreme_val, extreme_val]."""
+    return x / (1 + (x.square() / (extreme_val**2))).sqrt()
```

Comment:
```suggestion
def _softclip(x, bound=11.0):
    """Soft clipping function to keep x in [-bound, bound]."""
    return x * (1 + (x / bound).square()).rsqrt()
```

Any reason for using 11.0?